src/display/shared/columns/threads/LastPoster.js

```diff
@@ -12,6 +14,7 @@ export default (lastPosters, includeFilter) => ({
 					target="_blank"
 					rel="noopener noreferrer"
 					href={row.original.status && row.original.status.lastPostUrl}
+					onClick={stopPropagation}
```

Comment:
This prevents a click on the poster url, which opens a new tab, from having a row selection side effect